app/views/layouts/show_pdf.html.haml

```diff
@@ -4,7 +4,7 @@
   = render_quadicon(@record, :mode => :icon, :size => 72)
 - if @record.kind_of?(ConfigurationProfile)
   = render :partial => "#{controller}/main_configuration_profile"
-- elsif @record.kind_of?(ManageIQ::Providers::AutomationManager::InventoryGroup)
+- elsif @record.kind_of?(ManageIQ::Providers::ExternalAutomationManager::InventoryGroup)
```

Comment:
Is this also supposed to be `InventoryRootGroup`?  I don't really know what this file does, but we only refresh `InventoryRootGroup`s right now.